docs/data/base/components/snackbar/TransitionComponentSnackbar.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import { styled } from '@mui/system';
+import CheckRoundedIcon from '@mui/icons-material/CheckRounded';
+import CloseIcon from '@mui/icons-material/Close';
+import SnackbarUnstyled, { SnackbarCloseReason } from '@mui/base/SnackbarUnstyled';
+import Slide from '@mui/material/Slide';
```

Comment:
```suggestion
```

Please don't import components from `@mui/material`. I would rather see a transition component here implemented using `react-transition-group`. It can be very simpler implementation for sliding.

Reply:
Makes complete sense. I have done it now.